dagger.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "symfony",
+  "engineVersion": "v0.15.1",
```

Comment:
what would be the process to keep this up to date ? A bot opening a PR automatically or only if its fails then we need to upgrade.

Reply:
@Neirda24 hey! we will manually bump this every time we want to, or need to. Since I'm from Dagger, then I'll know exactly what important stuff is being released and if it's worth the upgrade or not